frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/GraphTableDialog.kt

```diff
@@ -42,7 +42,7 @@ class GraphTableDialog
     fun of(
         gitMacheteRepositorySnapshot: IGitMacheteRepositorySnapshot,
         windowTitle: String,
-        dimension: JBDimension = JBDimension(/* width */ 800, /* height */ 500),
```

Comment:
Any specific reason for that? ;)

Reply:
Oh ok, there used to be too much padding in the help/discover dialogs?

Reply:
yep. too much in case of discover and too small in case of help